sophia/src/term/test.rs

```diff
@@ -438,13 +426,14 @@ fn term_similar_but_not_eq() {
     let txt = "http://champin.net/#pa";
     let t1 = StaticTerm::new_iri(txt).unwrap();
     let t2 = StaticTerm::new_literal_dt(txt, xsd::anyURI).unwrap();
-    let t3 = StaticTerm::new_bnode(txt).unwrap();
+    // "http://champin.net/#pa" is not a valid blank node identifier
+    // let t3 = StaticTerm::new_bnode(txt).unwrap();
```

Comment:
Correct. I suggest changing txt to "foo" (which is a valid relative IRI ref), so that we can actually have similar terms of all categories.

Would you mind also adding t4 as a variable, for the sake of completeness?